.github/dependabot.yml

```diff
@@ -6,6 +6,13 @@ updates:
     interval: weekly
     day: sunday
     timezone: UCT
+  groups:
+    production-dependency-patches:
+      dependency-type: "production"
+      update-types: ["patch"]
```

Comment:
Here we group all patches together, but minor and major will get independent requests.

If we don't increase the schedule to "weekly", then there are less likely to be grouped, no?